### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by coderwithattitude**
recommended fixes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -763,10 +766,10 @@ <h2 class="s_title" data-before="about">PRE-SALE</h2>
...
-                                                <p>We have taken steps to modify our funding rounds to a “Pre-ico” or “Pre-sale” only efforts, cutting down on the time needed to market an “ICO” and to remain focused on development. Inadvertently the pre-ico only model will also reward only the most active participators with bonuses and benefits as we minimize the prominence of passive token holders in the organization. We need to reward the most active and earliest participators to reduce the influence of voting apathy among our crowd governed organization. </p>
+                                                <p>We have taken steps to modify our funding rounds to a “Pre-sale” or “Pre-sale” only efforts, cutting down on the time needed to market an “ICO” and to remain focused on development. Inadvertently the pre-sale only model will also reward only the most active participators with bonuses and benefits as we minimize the prominence of passive token holders in the organization. We need to reward the most active and earliest participators to reduce the influence of voting apathy among our crowd governed organization. </p>
```
Comment (adibas03) [inline index.html:769]: "Pre-Sale" repeated twice

**Review by adibas03 — APPROVED** [review]

File: js/myscript.js [hunk 2]
```diff
@@ -280,8 +289,17 @@ function initHighChart(){
                         y: 45,
                         sliced: true,
                         selected: true
-                    }, 
+                    },
                     {
+<<<<<<< HEAD
```
Comment (adibas03) [inline js/myscript.js]: un-resolved conflict

**Review by adibas03 — CHANGES_REQUESTED** [review]

**Comment by coderwithattitude** [comment]
@adibas03  kindly give final review

**Review by adibas03 — APPROVED** [review]
Looks good

**Pull request merged**